README.md

```diff
@@ -1 +1,81 @@
 # library-service-api
+
+Library API Service is a Django-based RESTful API for managing books borrowing and more. It optimizes the work of library administrators and make the service much more user-friendly.
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Features](#features)
+- [Installation](#installation)
+- [Endpoints](#endpoints)
+- [Presentation](#presentation)
+
+## Introduction
+
+Library API Service is designed to streamline the management of library-related data and user interactions.
+
+### Features:
+- Web-based
+- Manage books inventory
+- Manage books borrowing
+- Manage customers
+- Display notifications
+- Handle payments
+## Installation
+
+1. Clone the repository:
+
+   ```
+   git clone https://github.com/MarianKovalyshyn/library-service-api.git
+   ```
+2. Create .env file and define environmental variables following .env.example:
+   ```
+   POSTGRES_HOST= your db host
+   POSTGRES_DB= name of your db
+   POSTGRES_USER= username of your db user
+   POSTGRES_PASSWORD= your db password
```

Comment:
I would delete POSTGRES here because I am not sure what we will you it. Maybe we just leave sqlite